UnityProject/Assets/Scripts/Items/Cargo/Wrapping/WrappableObject.cs

```diff
@@ -61,11 +63,14 @@ private void FinishWrapping(WrappingPaper paper)
 					break;
 			}
 
-			result = Spawn.ServerPrefab(result, gameObject.AssumedWorldPosServer()).GameObject;
-			var wrap = result.GetComponent<WrappedObject>();
+			GameObject toSpawn;
+			toSpawn = Spawn.ServerPrefab(result, gameObject.AssumedWorldPosServer()).GameObject;
+			var wrap = toSpawn.GetComponent<WrappedObject>();
 			wrap.SetContent(gameObject);
-
+			GetComponent<ObjectContainer>().TransferObjectsTo(wrap);
+			GetComponent<PushPull>().parentContainer = wrap.GetComponent<PushPull>();
```

Comment:
These are pretty much the same component, could reduce these two to one component fetch, if you like.

Reply:
Sorry, I read ObjectContainer as ObjectBehaviour : /